app/src/main/java/com/sopt/now/compose/presentation/sign/SignScreen.kt

```diff
@@ -105,52 +130,17 @@ fun SignScreen() {
         // Nickname
         TextFieldWithTitle("닉네임", nickname, { newNickname ->
             nickname = newNickname
-            isNicknameValid = nickname.isNotBlank()
         }, "닉네임을 입력해주세요")
-        // Age
-        TextFieldWithTitle("나이", age, { newAge ->
-            age = newAge
-            isAgeValid = age.trim().length in MIN_AGE_LEN..MAX_AGE_LEN
-        }, "나이를 입력해주세요")
+        // Tel
+        TextFieldWithTitle("전화번호", tel, { newAge ->
```

Comment:
요런 string 값들도 strings.xml에서 관리해주면 좋을 것 같아요!
```suggestion
        TextFieldWithTitle(stringResource(id = R.stirng.~), tel, { newAge ->
```

Reply:
넵! 그렇게 적용해볼게요!